perfkitbenchmarker/data/cassandra/create-ycsb-table.cql.j2

```diff
@@ -12,10 +12,17 @@
  # See the License for the specific language governing permissions and
  # limitations under the License. #}
 {# Create a table for YCSB benchmark. #}
-create keyspace {{ keyspace }}
-  with placement_strategy = 'SimpleStrategy'
-  and strategy_options = {replication_factor: {{ replication_factor }} };
-
-use {{ keyspace }};
-
-create column family {{ column_family }};
+create keyspace ycsb WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': {{replication_factor}} };
+USE ycsb;
+create table usertable (
```

Comment:
YCSB itself will load tables with field count according to workload file ( https://github.com/brianfrankcooper/YCSB/blob/master/workloads/workload_template#L53) and we can also inject YCSB parameters to override YCSB workload file with ycsb_load_parameters and ycsb_run_parameters.